Jenkinsfile

```diff
@@ -64,37 +66,36 @@ pipeline {
         */
         stage('Test') {
           steps {
-            withEnvWrapper() {
-              unstash 'source'
-              unstash 'dotnet'
-              dir("${BASE_DIR}"){
-                sh '''#!/bin/bash
-                set -euxo pipefail
-                
-                # install tools
-                dotnet tool install -g dotnet-xunit-to-junit
-                for i in $(find . -name '*.??proj') 
-                do 
-                  dotnet add "$i" package XunitXml.TestLogger --version 2.0.0
-                  dotnet add "$i" package coverlet.msbuild
-                done
-                
-                # build
-                dotnet build
-                
-                # run tests
-                dotnet test -v n -r target -d target/diag.log --logger:"xunit" --no-build \
-                  /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura \
-                  /p:CoverletOutput=target/Coverage/ || echo -e "\033[31;49mTests FAILED\033[0m"
+            deleteDir()
+            unstash 'source'
+            unstash 'dotnet'
+            dir("${BASE_DIR}"){
+              sh '''#!/bin/bash
+              set -euxo pipefail
+              
+              # install tools
+              dotnet tool install -g dotnet-xunit-to-junit
+              for i in $(find . -name '*.??proj') 
+              do 
+                dotnet add "$i" package XunitXml.TestLogger --version 2.0.0
+                dotnet add "$i" package coverlet.msbuild
```

Comment:
Would be better if these dependencies were not hidden away in the jenkins file but embedded conditionally in the projects through `<PackageReference .... Condition="" />` so that CI can do e.g `dotnet test /p:UseCover=true /p:XmlLogger=true` and use those msbuild properties in the condition for the package reference. `dotnet test` already does a build so `dotnet build` is not needed.

Reply:
Those are packages used only on the CI, it is true that not be included in the project makes CI's scripts a little longer, if include those packages in the project does not affect to anything else, it would be really nice to have then inside the project.